developers-italia.oas.yaml

```
@@ -643,6 +643,13 @@ paths:
           in: query
           name: to
           description: Only logs before this time (RFC 3339 datetime)
+        - schema:
+            type: string
+            maxLength: 255
+            example: 'log'
```

I'd use something more practical here

```suggestion
            example: 'BAD publiccode'
```